src/components/organisms/PaneManager/PaneManagerSplitView.tsx

```diff
@@ -100,59 +101,72 @@ const PaneManagerSplitView: React.FC = () => {
   }, [leftActive]);
 
   return (
-    <S.SplitViewContainer ref={splitViewContainerRef} $gridTemplateColumns={splitViewGridTemplateColumns}>
-      {leftActive && leftMenuSelection && (
-        <S.LeftPaneContainer ref={leftPaneRef}>
+    <S.SplitViewContainer>
+      <ValidationPane height={paneHeight} />
+
+      <S.SplitViewGrid ref={splitViewContainerRef as any} $gridTemplateColumns={splitViewGridTemplateColumns}>
+        {leftActive && leftMenuSelection && (
+          <S.LeftPaneContainer ref={leftPaneRef}>
+            <ResizableBox
+              height={paneHeight}
+              width={splitViewContainerWidth * leftWidth}
+              minConstraints={[MIN_SPLIT_VIEW_PANE_WIDTH, paneHeight]}
+              maxConstraints={[leftPaneMaxWidth, paneHeight]}
+              axis="x"
+              resizeHandles={['e']}
+              handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
+                <span className="custom-modal-handle" ref={ref} />
+              )}
+              onResizeStop={resizeLeftPane}
+            >
+              <S.Pane id="LeftPane">
+                <Suspense fallback={null}>
+                  {leftMenuSelection === 'file-explorer' && <FileTreePane />}
+                  {leftMenuSelection === 'helm-pane' && <HelmPane />}
+                  {leftMenuSelection === 'kustomize-pane' && <KustomizePane />}
+                  {leftMenuSelection === 'templates-pane' && <TemplateManagerPane contentHeight={paneHeight} />}
+                </Suspense>
+              </S.Pane>
+            </ResizableBox>
+          </S.LeftPaneContainer>
+        )}
+
+        <S.Pane id="NavPane" $height={paneHeight} ref={navPaneRef}>
+          <NavigatorPane />
+        </S.Pane>
+
+        <S.EditorPaneContainer ref={editPaneRef}>
           <ResizableBox
             height={paneHeight}
-            width={splitViewContainerWidth * leftWidth}
+            width={splitViewContainerWidth * editWidth}
             minConstraints={[MIN_SPLIT_VIEW_PANE_WIDTH, paneHeight]}
-            maxConstraints={[leftPaneMaxWidth, paneHeight]}
+            maxConstraints={[editPaneMaxWidth, paneHeight]}
             axis="x"
-            resizeHandles={['e']}
+            resizeHandles={['w']}
```

Comment:
⚠️ I did not write this. Bad rebase? Validate!!

Reply:
Okay this is actually just `git diff` which is confused.

The old code is considered new see [New line 116](https://github.com/kubeshop/monokle/pull/1638/files#diff-05b1e9b8a3cfe46d620ca5f738a8c0953b312b06c7089d4697d76e1f31bae033R116) due to identation, then it moved the code below and saw it as this diff and saw the code below as deleted see [Old line 139](https://github.com/kubeshop/monokle/pull/1638/files#diff-05b1e9b8a3cfe46d620ca5f738a8c0953b312b06c7089d4697d76e1f31bae033L139).